hardinfo.1

```diff
@@ -0,0 +1,58 @@
+.\" Manpage for hardinfo.
+.\" https://github.com/lpereira/hardinfo/.
+.TH HARDINFO "1" "July 2017" "0.6" "User Commands"
+.SH NAME
+Hardinfo \- System profiler and benchmark for Linux systems.
+.SH USAGE
+hardinfo [OPTION...]
+.SH DESCRIPTION
+HardInfo is a system profiler and benchmark for Linux systems.
+
+HardInfo can gather information about your operating system 
```

Comment:
umm seems very large, i took the info from the help file in docs directory, that's why look as you see..

i'll change but think about it if are too large!?